sfv-validator/sfv-validator.py

```diff
@@ -81,7 +81,7 @@ def check_corruption(file, crc_hash):
         print(bcolors.FAIL + '> File ' + bcolors.WARNING + bcolors.BOLD + '\"' + file + '\"' + bcolors.ENDC + bcolors.FAIL + ' not found !' + bcolors.ENDC)
         return
 
-    if (crc(file) == crc_hash): print(bcolors.OKGREEN + '> The file ' + bcolors.OKBLUE  + bcolors.BOLD + '\"' +  file + '\"' + bcolors.ENDC + bcolors.OKGREEN + ' is not corrupted !' + bcolors.ENDC)
+    if (crc(file).lower() == crc_hash): print(bcolors.OKGREEN + '> The file ' + bcolors.OKBLUE  + bcolors.BOLD + '\"' +  file + '\"' + bcolors.ENDC + bcolors.OKGREEN + ' is not corrupted !' + bcolors.ENDC)
```

Comment:
The generated hash is always in uppercase, so if you want to lower you need to lower the two hash (the generated and the given one). 

```python
if (crc(file).lower() == crc_hash.lower()):
```

but i dont see the point in doing that as it is the same. though maybe i am mistaken..

Reply:
sorry, the title is borked :-\
i have found an .sfv-file which has lowercase crcs inside
currently the validator doesn't work in this case :-)

Reply:
mmh its weird the crcs should always be in uppercase as the "%X" in the crc generation function : 
```python  
crc32 = "%X" % (prev & 0xFFFFFFFF)
``` 
should forces the output to be in uppercase. could you share the file you used that generated the crc in lower ?

Reply:
i have found an .sfv-file FROM A DIFFERENT SOURCE which has lowercase CRCs inside :-)

Reply:
Oh Okay !! Now i understand thanks :) 
i will fix this when i have some time, so that the tool support lowercase crcs from generated from other sources.

I think that to fix this you only need to upper all element of the `arr_crc_hash` in the main..

Reply:
This should be okay now. Can you validate that this works correctly with your problematic file before i merge this branch pls
And thx for finding and contributing :)